src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1550,4 +1550,16 @@
   <data name="UseValidPlatformStringNoVersion" xml:space="preserve">
     <value>Version '{0}' is not valid for platform '{1}'. Do not use versions for this platform.</value>
   </data>
+  <data name="UseStringContainsCharOverloadWithSingleCharactersDescription" xml:space="preserve">
+    <value>'string.Contains(char)' is available as a better performing overload for single char lookup.</value>
+  </data>
+  <data name="UseStringContainsCharOverloadWithSingleCharactersMessage" xml:space="preserve">
+    <value>Consider using 'string.Contains(char)' instead of 'string.Contains(string)' when searching for a single character</value>
```

Comment:
Message should be more prescriptive.
```suggestion
    <value>Use 'string.Contains(char)' instead of 'string.Contains(string)' when searching for a single character</value>
```

Reply:
Please update.